tests/test_workflow.py

```diff
@@ -22,8 +22,12 @@
 
 
 CFG_FILE_BACKWARD = Path("tests/test_data/config_rhine.yaml")
+CFG_FILE_BACKWARD_PL = Path("tests/test_data/config_rhine_pl.yaml")
 CFG_FILE_FORWARD = Path("tests/test_data/config_west_africa.yaml")
 
+RTOL = 1e-5  # 0.001 %
+ATOL = 1e-7  # kg/m2/s. Approx 0.009 mm/day?
```

Comment:
@Peter9192 does this seem correct?

Reply:
I'm not sure. You use it both for preprocess (precip) and tracking (etrack/ptrack), do we expect similar values there? Also the tracking vars are accumulations since previous output, no? So no flux units.

Relative tolerance seems fine to me

Reply:
We can set different values for each. Absolute tolerance is necessary as otherwise near-0 values easily have large relative errors due to floating point inaccuracies.

Reply:
I was looking at the test verification files, these still had the old attributes so in #356 the values were updated of the regression test files, but not the attributes. I updated them now.

All variables we are looking at have the unit of `kg m-2`. This is 1 mm water depth equivalent. So an absolute tolerance of 1e-7 mm should be good enough here.